templates/careers/job-detail.html

```diff
@@ -80,4 +80,7 @@ <h3>Apply for this role</h3>
       </div>
     </div>
   </div>
+
+  <script async defer src="/static/js/file-validation.js"></script>
```

Comment:
You probably want versioned_static on these includes to cache bust.

Reply:
Oh yes, don't want to trip over that landmine again, lol